domains/blog.sanooj.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Sanoojes",
+        "email": "sanoojes6371@gmail.com"
+    },
+    "record": {
+        "CNAME": "sanoojes.github.io/blog"
```

Comment:
Remove /blog from the CNAME record

Reply:
its the project repo where am hosting my blog sanoojes.github.io is my personal portfolio

Reply:
> its the project repo where am hosting my blog sanoojes.github.io is my personal portfolio

if you want to connect it and for validations to not fail, just remove /blog since github can connect domains on any repos as long as you have sanoojes.github.io as the CNAME record